vms/avm/block/builder/builder_test.go

```diff
@@ -514,8 +517,22 @@ func TestBlockBuilderAddLocalTx(t *testing.T) {
 	}
 
 	baseDB := versiondb.New(memdb.New())
+	metadataDB := prefixdb.New([]byte("metadata"), baseDB)
```

Comment:
I was thinking all of non-state would live under this prefix... as an alternative we could just have each db live under their own prefix under the root db instead of sharing a common metadata prefix. I don't see a reason to do/not do one over the other so open to suggestions here